.github/workflows/auto_cherry_pick.yml

```diff
@@ -7,37 +7,18 @@ on:
       - closed
 
 jobs:
-  branch-matrix:
-    if: github.event.pull_request.merged == true && contains(github.event.pull_request.labels.*.name, 'CherryPick')
```

Comment:
Don't we need this condition now?

Reply:
We dont need branch matrix the condition still persists for autocherrypick job.